src/Fantomas/Context.fs

```diff
@@ -62,46 +62,46 @@ module WriterModel =
         | RestoreAtColumn c -> { m with AtColumn = c }
         | SetIndent c -> { m with Indent = c }
         | RestoreIndent c -> { m with Indent = c }
-        
+
     let updateAll cmds m = cmds |> Queue.fold (fun m c -> update c m) m
-    
+
 module WriterEvents =
     let normalize ev =
         match ev with
         | Write s when String.normalizeThenSplitNewLine s |> Array.length > 1 ->
             String.normalizeThenSplitNewLine s |> Seq.map (fun x -> [Write x]) |> Seq.reduce (fun x y -> x @ [ WriteLineInsideStringConst] @ y) |> Seq.toList
         | _ -> [ev]
-        
+
     let isMultiline evs =
         evs |> Queue.toSeq |> Seq.exists (function | WriteLine -> true | _ -> false)
 
-type internal Context = 
-    { Config : FormatConfig; 
+type internal Context =
+    { Config : FormatConfig;
       WriterModel : WriterModel
       WriterEvents : Queue<WriterEvent>;
       BreakLines : bool;
       BreakOn : string -> bool;
-      /// The original source string to query as a last resort 
-      Content : string; 
+      /// The original source string to query as a last resort
+      Content : string;
       /// Positions of new lines in the original source string
-      Positions : int []; 
+      Positions : int [];
```

Comment:
Sorry for touching so many lines, but the file was riddled with trailing whitespaces. It has to go.

Hopefully, if we start formatting the codebase itself with fantomas, there won't be any unrelated formatting fixes in the future! 😄